packages/site/src/data/react/courses.ts

```diff
@@ -400,4 +400,16 @@ export const courses: Course<typeof courseTags[number]>[] = [
 		href: "https://youtu.be/_kEDBitNbnY",
 		tags: ["graphQL", "data fetching", "faunaDB", "javascript marathon"],
 	},
+	{
+		title: "Build a Full-Stack App with Next.js, Supabase & Prisma",
+		author: "Grégory D'Angelo",
+		image: "https://github.com/alterclassio.png",
+		description:
+			"Learn to build a full-stack application from scratch using some of the best modern web technologies: React / Next.js, Prisma, and Supabase.",
+		paymentType: "free",
+		level: "begginer",
```

Comment:
Hey @gdangelo! Thank you for your contribution, just fix this little typo here before we can merge it!